docs/src/main/asciidoc/authentication.adoc

```diff
@@ -150,7 +150,7 @@ Following are the Azure built-in roles for authorizing access to Azure services
 
 |===
 
-When using Spring Cloud Azure Resource Manager to get the connection strings of Event Hubs, Service Bus, and Storage Queue, and properties of Cache for Redis, assign the Azure built-in role `Contributor`. Azure Cache for Redis is special, and you can also assign the `Redis Cache Contributor` role to get the Redis properties.
+NOTE: When using Spring Cloud Azure Resource Manager to get the connection strings of Event Hubs, Service Bus, and Storage Queue, and properties of Cache for Redis, assign the Azure built-in role `Contributor`. Azure Cache for Redis is special, and you can also assign the `Redis Cache Contributor` role to get the Redis properties.
```

Comment:
get the connection strings of Event Hubs, Service Bus, and Storage Queue, **or** properties of Cache for Redis

Reply:
Thanks.